docs/data/toolpad/core/components/dashboard-layout/dashboard-layout.md

```diff
@@ -146,3 +146,7 @@ This allows you to add, for example:
 - footer content in the sidebar.
 
 {{"demo": "DashboardLayoutSlots.js", "height": 400, "iframe": true}}
+
+Through this, you can also modify the position of the `<Account />` component and use it inside the sidebar:
```

Comment:
In https://github.com/mui/toolpad/pull/4340 I'm adding an **Examples** section here, I guess that we can just have this under that section once it gets added.